src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -245,6 +250,21 @@ const SearchFieldList = ({ onSearch }: Props) => {
     return () => window.removeEventListener('popstate', readSearchParamsFromURL);
   }, [initialSearchParams]);
 
+  const rename = () => {
+    let dep = departments;
+    dep.map((d, i) => {
+      if (/^Office of /.test(d)) {
+        dep[i] = dep[i].replace(/^Office of /, '') + ' (Office of)';
+      } else if (/^Center for /.test(d)) {
+        dep[i] = dep[i].replace(/^Center for /, '') + ' (Center for)';
+      } else if (/^Department of /.test(d)) {
+        dep[i] = dep[i].replace(/^Department of /, '') + ' (Department of)';
+      }
+    });
+    console.log(dep);
+    return dep;
+  };
```

Comment:
Wouldn't it make more sense to do this within the `peopleSearchService`? It seems to me that the service code, which handles retrieving and formatting code, should be the single place where this transformation from internal form to user interface form happens.

Reply:
That is right, Thank you!